commands/show-long-connections.go

```diff
@@ -51,13 +48,13 @@ func init() {
 			}
 
 			if c.Bool("human-readable") {
-				err := showConnsHuman(data)
+				err := showConnsHuman(data, c.Bool("network-names"))
```

Comment:
There are a lot of places where we are using the same hardcoded strings. We might want to consider declaring these somewhere as constants for use throughout the program.

Reply:
`goconst` should be checking these. I didn't realize it was already enabled. We may need to investigate why it's not reporting them and decide if we should tweak its logic.
https://github.com/activecm/rita/blob/master/.golangci.yml#L8
I'll make an issue for this.

Reply:
Agreed, we should probably spin this off as a separate tech-debt PR. Our other projects need this as well.